### PR TITLE
set meshconfig.sdsUseK8sSaJwt through sds helm value

### PR DESCRIPTION
**Pull request by quanjielin**
https://github.com/istio/istio/issues/9035

if ```sds.useK8sSATrustworthyJwt``` is set, use trustworthy jwt to request key/cert, which is the preferred way but only avaiable in k8s 1.12 or later; fallback to if ```sds.useK8sSANormalJwt``` is set, use normal k8s sa jwt to request key/cert.

also do some renaming in this pr

### CODE REVIEW

File: install/kubernetes/helm/istio-remote/templates/sidecar-injector-configmap.yaml [hunk 1]
```diff
@@ -170,7 +170,7 @@ data:
             - NET_ADMIN
           runAsGroup: 1337
           {{ "[[ else -]]" }}
-          {{ if and .Values.global.sds.enabled .Values.global.sds.enableTokenMount }}
+          {{ if and .Values.global.sds.enabled .Values.global.sds.useK8sSATrustworthyJwt }}
```
Comment (andraxylia) [inline install/kubernetes/helm/istio-remote/templates/sidecar-injector-configmap.yaml]: For a better UX, consider using a shorter and less typo-prone name, like useTrustworthyJwt, or useJwt (if all Jwt are trustworty).

File: install/kubernetes/helm/istio-remote/values.yaml [hunk 2]
```diff
@@ -258,7 +258,8 @@ global:
   sds:
     enabled: false
     udsPath: ""
-    enableTokenMount: false
+    useK8sSATrustworthyJwt: false
+    useK8sSANormalJwt: false
```
Comment (andraxylia) [inline install/kubernetes/helm/istio-remote/values.yaml]: Nit again: so a normal Jwt is not to be trusted?

Maybe it's verified Jwt?

File: install/kubernetes/helm/istio-remote/templates/sidecar-injector-configmap.yaml (hunk 1 above)
Comment (quanjielin) [reply install/kubernetes/helm/istio-remote/templates/sidecar-injector-configmap.yaml]: renamed.

File: install/kubernetes/helm/istio-remote/values.yaml (hunk 2 above)
Comment (quanjielin) [reply install/kubernetes/helm/istio-remote/values.yaml]: the normal k8s sa jwt could be verified against k8s api server, just the trustworthy just has more claims like expired time.

**Pull request merged**